example/datastack/README.md

```diff
@@ -0,0 +1,58 @@
+<!-- markdownlint-disable MD041 -->
+<!-- markdownlint-disable MD028 -->
+<p align="center"><img width="100px" src="https://www.svgrepo.com/show/12675/bar-chart.svg" align="center" alt="Vagrant-hashistack" /><h2 align="center">Datastack with Redash</h2></p>
+
+## Content
+1. [Introduction](#introduction)
+2. [Requirements](#requirements)
+3. [Starting Datastack](#starting-datastack)
+4. [A note about MinIO](#a-note-about-minio)
+5. [Access Redash, MinIO and Presto](#access-redash-minio-and-presto)
+6. [Consul Connect and Proxies](#consul-connect-and-proxies)
+
+## Introduction
+This example provisions a full datastack.
+
+Provisioned stack:
+- [MinIO](https://min.io/) (storage)
+- [Presto](https://prestodb.io/) (SQL-engine)
+- [Redash](https://redash.io/) (Interface for writing SQL to Presto)
+
+## Requirements
+See [prerequisites](../../template_README.md#install-prerequisites)
+
+### Starting Datastack
+Run
+```makefile
+make up
+```
+in your terminal to provision the datastack.
+
+### A note about MinIO
+This example will set up _two_ MinIOs. There is one bundled with the vagrant-box that is used by the developers of this repository for _development_. In addition to this, this example sets up another MinIO that is intended to use together with Redash and Presto. Read [Access Redash, MinIO and Presto](#access-redash-minio-and-presto) on how to connect to the latter.
+
+
+### Access Redash, MinIO and Presto
+When provisioning of the example is finished you can then open up access to Redash, Presto, and MinIO by running
+```makefile
+make connect-to-all
+```
+
+> :warning: [gnome-terminal](https://help.gnome.org/users/gnome-terminal/stable/) is required for the command above. If you do not have gnome-terminal you can run `make proxy-redash`, `make proxy-presto`, and `make proxy-presto` in three separate terminal windows to achieve the same as `make connect-to-all`.
```

Comment:
If there is a similar feature for Mac OS to open multiple terminals we can do a simple check in `connect-to-all` with something like:
```makefile
OS = $(shell uname)
lol:
ifeq ($(OS), Linux)
	 echo OS is Linux: $(OS)
ifeq ($(OS), Darwin)
	 echo OS is MacOS: $(OS)
else
	 # do something else
endif
```

Reply:
I agree, could make an issue on that? :D

Reply:
Sure 😉

Reply:
Nice!